src/Sylius/Bundle/AdminBundle/Menu/OrderShowMenuBuilder.php

```diff
@@ -40,18 +41,20 @@
     private $stateMachineFactory;
 
     /**
-     * @param FactoryInterface $factory
-     * @param EventDispatcherInterface $eventDispatcher
-     * @param StateMachineFactoryInterface $stateMachineFactory,
+     * @var CsrfTokenManagerInterface
      */
+    private $csrfTokenManager;
+
     public function __construct(
         FactoryInterface $factory,
         EventDispatcherInterface $eventDispatcher,
-        StateMachineFactoryInterface $stateMachineFactory
+        StateMachineFactoryInterface $stateMachineFactory,
+        CsrfTokenManagerInterface $csrfTokenManager
```

Comment:
It's a security-related BC break, will be documented in UPGRADEs.md before the release.